content/docs/setup/kubernetes/install/kubernetes/index.md

```diff
@@ -137,6 +146,12 @@ $ kubectl apply -f install/kubernetes/istio-demo-auth.yaml
     prometheus-67cdb66cbb-9w2hm                                    1/1     Running     0          1m
     {{< /text >}}
 
+1.  Run the following command to verify the installation succeeds:
+
+    {{< text bash >}}
+    $ istioctl verify-install -f install/kubernetes/istio-demo.yaml
```

Comment:
Does user need to do the previous 2 steps and also this? Isn't this sufficient to verify?

Reply:
there are some tips about how to access the gateway in the previous 2 steps. And these two additional check do not harm. So can we keep it here?

Reply:
I put the short output here as the verbose version will have really long output.